netbox/dcim/choices.py

```diff
@@ -345,6 +345,10 @@ class PowerPortTypeChoices(ChoiceSet):
     TYPE_DC = 'dc-terminal'
     # Proprietary
     TYPE_SAF_D_GRID = 'saf-d-grid'
+    TYPE_NEUTRIK_POWERCON = 'neutrik-powercon'
+    TYPE_NEUTRIK_POWERCON_32A = 'neutrik-powercon-32'
+    TYPE_NEUTRIK_TRUE1 = 'neutrik-true1'
+    TYPE_NEUTRIK_TRUE1_TOP = 'neutrik-true1-top'
```

Comment:
The types listed [here](https://www.neutrik.com/en/products/power) are:

- powerCON 20A
- powerCON 32A
- powerCON TRUE1
- powerCON TRUE1 TOP

We should add exactly these to NetBox, with the same exact capitalization and noting the amperage where indicated. Also be sure that the constant value includes "POWERCON" in its name as that is part of the identifier (e.g. `TYPE_NEUTRIK_POWERCON_TRUE1`).

```suggestion
    TYPE_NEUTRIK_POWERCON_20A = 'neutrik-powercon-20a'
    TYPE_NEUTRIK_POWERCON_32A = 'neutrik-powercon-32a'
    TYPE_NEUTRIK_POWERCON_TRUE1 = 'neutrik-powercon-true1'
    TYPE_NEUTRIK_POWERCON_TRUE1_TOP = 'neutrik-powercon-true1-top'
```